src/ingest-pipeline/airflow/plugins/globus_auth/globus_auth.py

```diff
@@ -114,15 +114,15 @@ def login(self, session=None):
         try:
             if 'code' not in request.args:
                 auth_uri = self.globus_oauth.oauth2_get_authorize_url(additional_params={
-                    "scope": "openid profile email urn:globus:auth:scope:transfer.api.globus.org:all urn:globus:auth:scope:auth.globus.org:view_identities urn:globus:auth:scope:nexus.api.globus.org:groups"})
+                    "scope": "openid profile email urn:globus:auth:scope:transfer.api.globus.org:all urn:globus:auth:scope:auth.globus.org:view_identities urn:globus:auth:scope:groups.api.globus.org:groups"})
```

Comment:
@jpuerto-psc The scope for Groups API that I use here is `urn:globus:auth:scope:groups.api.globus.org:all` which is a bit different from what you're specifying.